src/python/pants/backend/codegen/protobuf/python/rules_integration_test.py

```diff
@@ -271,3 +274,88 @@ def test_grpc(rule_runner: RuleRunner) -> None:
         source_roots=["src/protobuf"],
         expected_files=["src/protobuf/dir1/f_pb2.py", "src/protobuf/dir1/f_pb2_grpc.py"],
     )
+
+
+def test_grpc_mypy_plugin(rule_runner: RuleRunner) -> None:
+    rule_runner.create_file(
+        "src/protobuf/dir1/f.proto",
+        dedent(
+            """\
+            syntax = "proto3";
```

Comment:
It looks like this stanza is repeated a few times in the file now and could be factored up into a module-level constant. No worries if you don't get to it before we merge, I can fix in the followup to get `--pex-path` working.